webplugin/js/app/kommunicate-ui.js

```diff
@@ -600,8 +601,8 @@ handleAttachmentIconVisibility : function(enableAttachment, msg, groupReloaded)
                 success: function (res) {
                     if(res.status === "success"){
                     WAITING_QUEUE = res.response;
-                    document.getElementById('waiting-queue-number') && (document.getElementById('waiting-queue-number').innerHTML = "#" + parseInt(WAITING_QUEUE.indexOf(parseInt(groupId)) + 1));
-                    if (waitingStatus) {
+                    if (waitingStatus && WAITING_QUEUE.length) {
+                        document.getElementById('waiting-queue-number') && (document.getElementById('waiting-queue-number').innerHTML = "#" + parseInt(WAITING_QUEUE.indexOf(parseInt(groupId)) + 1));
```

Comment:
I know this is old code but we can store "document.getElementById('waiting-queue-number')" in a variable rather than calling it multiple time.